targets/targets.json

```diff
@@ -3981,5 +3981,24 @@
         "detect_code": ["7013"],
         "release_versions": ["5"],
         "bootloader_supported": true
+    },
+    "RDA5981X": {
+        "inherits": ["Target"],
+        "core": "Cortex-M4F",
+        "extra_labels": ["RDA", "UNO_91H", "FLASH_CMSIS_ALGO"],
+        "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
+        "macros": ["MBED_HEAP_STATS_ENABLED", "TWO_RAM_REGIONS"],
+        "device_has": ["PORTIN", "PORTOUT", "PORTINOUT", "INTERRUPTIN", "SERIAL", "STDIO_MESSAGES", "PWMOUT", "SPI", "LOWPOWERTIMER", "ANALOGIN", "FLASH", "RTC"],
+        "post_binary_hook": {"function": "RDATargetCode.binary_hook"},
+        "release_versions": ["2", "5"]
+    },
+    "UNO_81C": {
+        "inherits": ["RDA5981X"],
+        "detect_code": ["8002"]
+    },
+    "UNO_91H": {
+        "inherits": ["UNO_81C"],
```

Comment:
This should inherit from "RDA5981X" (parent target) rather than another derived target (UNO_81C).. this is required for Mbed 2 to build from online compiler.. I can see release_version above needs Mbed 2 as well..

Reply:
RDA5981X chip series contain 81C/81AM/81A, they share the original name "91H". especially, 91H is an alias for 81C. 
UNO_91H will be removed to meet the mbed 2 rules.